addons/web/static/src/legacy/js/core/menu.js

```diff
@@ -0,0 +1,160 @@
+/** @odoo-module alias=web.AutoMoreMenu **/
+
+export default {
```

Comment:
Instead of exporting an object with the function, export the two functions individually? You can then import them the same way you already did if I am not mistaken.

Reply:
Same